modules/users/server/routes/admin.server.routes.js

```diff
@@ -18,8 +18,9 @@ module.exports = function (app) {
   app.route('/api/users/:userId')
     .get(adminPolicy.isAllowed, admin.read)
     .put(adminPolicy.isAllowed, admin.update)
```

Comment:
Oh, I was confused for a second because you didn't highlight the right code for it but yeah when I look at the commits I see what you mean. Will address.